robottelo/hosts.py

```diff
@@ -1657,21 +1719,6 @@ def get_features(self):
         """Get capsule features"""
         return requests.get(f'https://{self.hostname}:9090/features', verify=False).text
 
-    def enable_capsule_downstream_repos(self):
-        """Enable CDN repos and capsule downstream repos on Capsule Host"""
-        # CDN Repos
-        self.register_to_cdn()
-        for repo in getattr(constants, f"OHSNAP_RHEL{self.os_version.major}_REPOS"):
-            result = self.enable_repo(repo, force=True)
-            if result.status:
-                raise CapsuleHostError(f'Repo enable at capsule host failed\n{result.stdout}')
-        # Downstream Capsule specific Repos
-        self.download_repofile(
-            product='capsule',
-            release=settings.capsule.version.release,
-            snap=settings.capsule.version.snap,
-        )
-
     def capsule_setup(self, sat_host=None, capsule_cert_opts=None, **installer_kwargs):
         """Prepare the host and run the capsule installer"""
```

Comment:
@rmynar Do you plan to update the `capsule_setup` as well? Could the new helpers be used in it?